src/cli.ts

```diff
@@ -208,6 +217,12 @@ export async function importPath(
     const deps = parseDeps(await fs.readFile(filepath))
     await installDeps(deps, dir)
   }
+
```

Comment:
Let's just modify the current flow:
```js
if (argv.install) {
    const deps = parseDeps(await fs.readFile(filepath))
    await installDeps(deps, dir, argv['install-registry'])
}
```

Reply:
Are you suggesting to make such a flow? Maybe then replace --install-registry with --registry as in npm?

```bash
zx --install --registry='https://npm-proxy.example.com' script.mjs
```

Reply:
ok

Reply:
d1